.drone.star

```diff
@@ -354,28 +371,46 @@ def acceptance():
 								installNPM() +
 								buildPhoenix() +
 								installCore(server, db) +
-								cloneOauth() +
-								setupServerAndApp(params['logLevel']) +
-								owncloudLog() +
-								fixPermissions() +
 								(
-									installFederatedServer(federatedServerVersion, db, federationDbSuffix) +
-									setupFedServerAndApp(params['logLevel']) +
-									fixPermissionsFederated() +
-									owncloudLogFederated() if params['federatedServerNeeded'] else []
+									(
+										cloneOauth() +
+										setupServerAndApp(params['logLevel']) +
+										owncloudLog() +
+										fixPermissions() +
+										(
+											installFederatedServer(federatedServerVersion, db, federationDbSuffix) +
+											setupFedServerAndApp(params['logLevel']) +
+											fixPermissionsFederated() +
+											owncloudLogFederated() if params['federatedServerNeeded'] else []
+										)
+									) if not params['runningOnOCIS'] else (
+										buildKonnectd() +
+										buildOcis() +
+										buildReva() +
+										konnectdService() +
+										revaService() +
+										ocisServices() +
+										redisService()
+									)
 								) +
 								copyFilesForUpload() +
-								runWebuiAcceptanceTests(suite, alternateSuiteName, params['filterTags'], params['extraEnvironment'], browser),
+								runWebuiAcceptanceTests(suite, alternateSuiteName, params['filterTags'], params['extraEnvironment'], browser, params['runningOnOCIS']),
 							'services':
-								phoenixService() +
-								owncloudService() +
 								(
-									owncloudFederatedService() +
-									databaseServiceForFederation(db, federationDbSuffix)
-									if params['federatedServerNeeded'] else []
-								) +
-								browserService(alternateSuiteName, browser) +
-								databaseService(db),
+									browserService(alternateSuiteName, browser) +
+									databaseService(db) +
+									phoenixService() +
+									owncloudService() +
+									(
+										owncloudFederatedService() +
+										databaseServiceForFederation(db, federationDbSuffix)
+										if params['federatedServerNeeded'] else []
+									)
+								) if not params['runningOnOCIS'] else (
+									browserService(alternateSuiteName, browser) +
+									databaseService(db) +
```

Comment:
`browserService` and `databaseService` are duplicated in the `if` and `else`. Those could be put outside this if-else.